### ISSUE TITLE
Doubts about start

### ISSUE THREAD

**Issue by fbraz** [bubble-dev/_#68]
[https://github.com/deepsweet/start](https://github.com/deepsweet/start) seemed like a very interesting project, but now it's archived and moved into a strange monorepo. I thought that monorepos were intented for related projects but the projects inside bubble-dev/_ aren't related. I don't understand why start was archived. Wouldn't that prevent adoption?

**Comment by deepsweet** [comment]
Hi.

I hope this strange monorepo will turn into a very good place someday, and it's the only way for Start to live because I almost don't have free time for projects like that and nobody has shown any signs of really interested. It's a very niche thing, task runner in 2019. Here I can work on Start with related and deeply connected projects almost every day, that's the place I put in a lot of effort recently.

So it's a good move, not like Start is going to be deprecated, but quite the opposite. I'll add such explanation to the original repo readme.

**Comment by xaviervia** [comment]
Yes, exactly. First of all thanks for asking the question, it made us realize that we need to explain this better in the READMEs, we had missed that. 

About Start in particular, the goal of moving to this repo is to be able to grow the things more, and we are a small group collaborating in these now, so hopefully it will have even more support, since we can do more documentation and support as a group. 

About the strange monorepo, we are trying it out, calling it a _metarepo_, it is a new concept that we are experimenting with essentially it is to simplify the development process (many of the projects are dependencies of each other, so we can iterate in all at the same time, and with `@auto`  tools dependency management for upgrading versions is automatic). Even when they are not cross dependencies, they are projects that we tend to work on several at the same time. It is clear that we need to explain better what metarepo is, we are working on adding more documentation, the first step was simply moving things in, now we are starting to work more intently on docs.

**Comment by fbraz** [comment]
I understand it a bit more now. I discovered Start this monday. I never heard of it before, it deserves more exposure, like an article on medium or something. 

I'm not sure if I understand it well though. The project caught my attention because I tought manage babel, webpack, the typescript compiler and jest with Start, so I could get rid of scattered config files and my package.json would be easier to understand and manage, and my root directory would be cleaner.
Is this reasoning ok?

Besides, 
If I install start with a preset like this:
```bash
yarn add --dev @deepsweet/start @deepsweet/start-preset-node-ts-lib
```
Does it mean I don't need to have a babel and webpack installed as dev-dependencies?

**Comment by deepsweet** [comment]
>so I could get rid of scattered config files and my package.json would be easier to understand and manage, and my root directory would be cleaner.

That's the main goal of Start "presets" – once you have crafted your first "task file" you can easily extract it into a separate package with all its dependencies and then reuse in all your projects. [Usage example](https://github.com/deepsweet/dleet/blob/81a548395c0d6451078638f8e4bbef76acd7ac93/package.json#L20) of my personal `@deepsweet` preset – all I need to get all my commands like `build`, `test`, `lint`, etc is to install a single preset package.

We've been [using Start presets](https://github.com/bubble-dev/_/blob/master/packages/bubble-dev/start-preset/src/index.ts) quite heavily.

>Does it mean I don't need to have babel and webpack installed as dev-dependencies

Basically – yes. There are some minor things to keep in mind, like [`require.resolve`](https://github.com/deepsweet/_/blob/283d9c32dfbdf21d5b344d4f956df55dd4cf616e/packages/start-preset-node-ts-lib/src/index.ts#L23) in configs like Babel or Webpack, peer dependencies warnings and such, but that's another question.

**Comment by fbraz** [comment]
I will try Start later on today to see if I can recreate my current config files and stuff. I will close this. Thank you guys
